README.md

```diff
@@ -114,6 +114,41 @@ JSONSchemer.schema(
 )
 ```
 
+## Parsing errors
+To get a list of errors map the results like this.
+``` 
+require 'pp'
+
+schemer = JSONSchemer.schema(your-json-schema)
+
+result = schemer.validate({
+	Values you're inputting
+})
+
+# use pp to make it nice to look at in your console.
+pp result.map { |error| error.slice('data', 'data_pointer', 'type') }
+
+```
+
+## Local file references
```

Comment:
Maybe we should do a more general description of `ref_resolver` here. The default depends on what kind of object you pass in as a schema. `Pathname` will use a local file resolver. Other types will raise `JSONSchemer::UnknownRef` by default. There's also a special `'net/http'` value that uses `Net::HTTP.get`. Otherwise you'll need to pass a callable (proc/lambda) object to `ref_resolver` that receives a `URI` and returns the subschema as a hash.